spec/support/e2e_setup.rb

```diff
@@ -12,11 +12,14 @@
   def sign_in(user, password: DEFAULT_USER_PASSWORD)
     # this should go into before-each but that seems to hang up some tests
     visit('/')
-    disable_transitions
 
     fill_in 'Email Address', with: user.email
     fill_in 'Password', with: password
     click_button('Sign In')
+    assert_text user.full_name # user's name should appear in the header
+
+    # page.driver.refresh # https://github.com/rubycdp/cuprite/issues/219
```

Comment:
lets wait and see if https://github.com/greenriver/hmis-warehouse/pull/4735 fixes things. I couldn't yet reproduce the cuprite error after that. If we do see it again, we can try this